rustls-post-quantum/src/lib.rs

```diff
@@ -59,12 +61,21 @@ use rustls::crypto::{
 use rustls::ffdhe_groups::FfdheGroup;
 use rustls::{Error, NamedGroup, PeerMisbehaved, ProtocolVersion};
 
-/// A `CryptoProvider` which includes `X25519MLKEM768` key exchange.
+mod mlkem;
+
+pub use mlkem::MLKEM768;
+
+/// A `CryptoProvider` which includes `X25519MLKEM768` and `MLKEM768`
+/// key exchanges.
 pub fn provider() -> CryptoProvider {
     let mut parent = default_provider();
+
     parent
         .kx_groups
         .insert(0, &X25519MLKEM768);
+
+    parent.kx_groups.insert(1, &MLKEM768);
```

Comment:
feels a bit tidier as:
```rust
    parent
        .kx_groups
        .splice(0..0, [X25519MLKEM768, MLKEM768]);
```

Reply:
Agreed, though I think we can drop the left `0` in `0..0`.